service/controller/v13/version_bundle.go

```diff
@@ -22,6 +22,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Added CA public key into trusted user keys for SSO ssh.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "k8s-kvm",
```

Comment:
Is this the right name? Discuss plz.

Reply:
Something like `k8s-kvm-docker-image` would be more precise and fairly obvious to me if I was a customer.
But we did name the repo just `k8s-kvm` so I am unsure here as well.

Reply:
I have zero opinion on this. If anyone else from @giantswarm/sig-operator has one, I'd be happy to hear it.